.azure/asgardeo-java-oidc-sdk-sca-scan.yaml

```diff
@@ -45,3 +45,10 @@ jobs:
         parameters:
           API_KEY: $(FOSSA-API-KEY)
           BRANCH: master
+      - template: ci-pipelines/templates/sca-scan-jfrog.yaml@templates
+        parameters:
+          PROJECT_TYPE: mvn
+          GITHUB_CONNECTION: $(GITHUB_CONNECTION) # GitHub connection name to Show the vulnerability report as a PR comment
```

Comment:
This has to be GITHUB_CONNECTION: asgardeo-github-sca-scan

Reply:
Changed in commit 8b4a23d611b53927fcd8025dc07582f343997bd3